action.yml

```diff
@@ -48,7 +48,7 @@ runs:
             # v0.6.0's linux release has a dependency on `libpcsclite1`
             sudo apt-get update -q
```

Comment:
This line is a bit scary :S. What about checking if that dependency exists before running the `update` ?

Reply:
We could tackle this in a different PR, it is unrelated to your changes here. Just sharing my thoughts here :).